src/api/server_server.rs

```diff
@@ -627,7 +627,7 @@ pub async fn get_server_version_route(
 
     Ok(get_server_version::v1::Response {
         server: Some(get_server_version::v1::Server {
-            name: Some(env!("CARGO_PKG_NAME").to_owned()),
+            name: Some(env!("Conduwuit").to_owned()),
```

Comment:
This needs to actually be:

```suggestion
            name: Some("Conduwuit".to_owned()),
```

`env!` is a macro to get an environment variable, so that would be looking for an environment variable named `Conduwuit`

Reply:
Ah. makes sense.